lib/Toolbar.js

```diff
@@ -56,7 +56,6 @@ export default class Toolbar extends Component {
         actions: PropTypes.arrayOf(PropTypes.shape({
             icon: PropTypes.string.isRequired,
             onPress: PropTypes.func,
-            counter: PropTypes.shape()
```

Comment:
You're right. Need to refactor this library :/ Thank you Vojta, I'll merge it ;) 👍